app/src/main/java/com/stl/skipthelibrary/BorrowersBooksActivity.java

```diff
@@ -16,5 +21,34 @@ protected void onCreate(Bundle savedInstanceState) {
         BottomNavigationView navigation = findViewById(R.id.bottom_navigation);
         navigation.setOnNavigationItemSelectedListener(new NavigationHandler(this));
         navigation.setSelectedItemId(R.id.borrow);
+
+        getBooks();
+        initRecyclerView();
+    }
+
+    private void getBooks() {
+        //Currently just test data as firebase is empty
+        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+                "test author", new Rating()),"testUsername",
+                new RequestHandler(new State(BookStatus.ACCEPTED,null, null)), null, new Rating()));
+        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+                "test author", new Rating()),"testUsername",
+                new RequestHandler(new State(BookStatus.REQUESTED,null, null)), null, new Rating()));
+        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+                "test author", new Rating()),"testUsername",
+                new RequestHandler(new State(BookStatus.AVAILABLE,null, null)), null, new Rating()));
+        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+                "test author", new Rating()),"testUsername",
+                new RequestHandler(new State(BookStatus.BORROWED,null, null)), null, new Rating()));
+        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+                "test author", new Rating()),"testUsername",
+                new RequestHandler(new State(BookStatus.REQUESTED,null, null)), null, new Rating()));
+    }
+
+    private void initRecyclerView(){
+        RecyclerView recyclerView = findViewById(R.id.borrowerBookRecyclerView);
```

Comment:
@tymoorej just again having this as a activity variable may be a good thing, just allows for future use